src/zenml/enums.py

```diff
@@ -327,3 +327,11 @@ class ResponseUpdateStrategy(StrEnum):
     ALLOW = "allow"
     IGNORE = "ignore"
     DENY = "deny"
+
+
+class MetadataResourceTypes(StrEnum):
+    """All possible resource types for adding metadata."""
+
+    PIPELINE_RUN = "pipeline_run"
+    STEP_RUN = "step_run"
+    ARTIFACT = "artifact"
```

Comment:
Similar to the previous comment, this will be affected by the [PR here](https://github.com/zenml-io/zenml/pull/2081).